src/ErrorPage.php

```diff
@@ -426,6 +435,13 @@ protected function getCodes()
             510 => _t('SilverStripe\\ErrorPage\\ErrorPage.CODE_510', '510 - Not Extended'),
             511 => _t('SilverStripe\\ErrorPage\\ErrorPage.CODE_511', '511 - Network Authentication Required'),
         ];
+
+        $allowedCodes = self::config()->get('allowed_error_codes');
```

Comment:
```suggestion
        $allowedCodes = static::config()->get('allowed_error_codes');
```
Preferably only use `self` if we're using uninherited config. In this case someone could have some crazy use case where they want a sublclass of `ErrorPage` to have different allowed codes, so using `static` enables that use case.